ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/api/AemApi.java

```diff
@@ -31,17 +30,19 @@
     Call<ResponseBody> downloadResource(@Url Uri uri);
 
     static AemApi buildInstance(@NonNull final String uri) {
-        // create OkHttp client
-        final OkHttpClient.Builder builder = new OkHttpClient.Builder()
-                .connectTimeout(60, TimeUnit.SECONDS)
-                .readTimeout(60, TimeUnit.SECONDS);
```

Comment:
How I imagine this actually working is:
```java
final SSLSocketFactory sslSocketFactory = DynamicSSLSocketFactory.builder()
        .addEnabledProtocols("TLSv1.2")
        .build();

final OkHttpClient.Builder builder = new OkHttpClient.Builder()
        .connectTimeout(60, TimeUnit.SECONDS)
        .readTimeout(60, TimeUnit.SECONDS)
        .sslSocketFactory(sslSocketFactory);
```

Reply:
I know `sslSocketFactory(SSLSocketFactory)` is deprecated, but I'm fine with using it for now because it pulls the stock TrustManager automatically.